tests/e2e/helpers/async-helper.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Helper function to work with arr.filter() method with async functions
+ * @param array The array
+ * @param callback The callback function need to be processed
+ */
+async function asyncFilter(array: any, callback: any) {
```

Comment:
Couldn't you declare a dedicated type these both parameters?

Reply:
fixed, thanks